ddtrace/llmobs/_writer.py

```diff
@@ -543,17 +564,39 @@ def dataset_bulk_upload(self, dataset_id: str, records: List[DatasetRecord]):
             raise ValueError(f"Failed to upload dataset from file: {resp.status} {resp.get_json()}")
         logger.debug("successfully uploaded with code %d", resp.status)
 
-    def project_create_or_get(self, name: str) -> str:
+    def project_create_or_get(self, name: Optional[str] = None) -> Project:
+        default_project_name = self._default_project.get("name", "")
```

Comment:
nit: a little weird to have a default value in the `.get()` on the default project since there has to be a name according to the type. It also becomes a 3th definition of the the default project name

Reply:
ok then no `get` at all